modules/wysiwyg.php

```diff
@@ -12,13 +12,14 @@ function wysiwyg_d7csp_hosts() {
   $hosts = [];
   wysiwyg_initialize_cache();
   $cache = cache_get('wysiwyg_css');
-  foreach (list_themes() as $theme) {
-    foreach (($cache->data[$theme->name]['files'] ?? []) as $url) {
-      $parts = parse_url($url);
-      if ($parts['host'] && $parts['host'] != $_SERVER['HTTP_HOST']) {
-        $hosts['style-src'][] = $parts['scheme'] . '://' . $parts['host'];
-        $hosts['font-src'][] = $parts['scheme'] . '://' . $parts['host'];
-        $hosts['img-src'][] = $parts['scheme'] . '://' . $parts['host'];
+  $themes = list_themes();
+  foreach (array_keys($cache->data) as $theme_name) {
```

Comment:
This seems to throw errors, when there is nothing found in the cache. Not sure exactly when this happens as `wysiwyg_initialize_cache()` is called right before it.